packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -108,7 +108,19 @@ export const createHandlesTracker = (
       const filteredUtxo = utxo.flatMap(([_, txOut]) =>
         uniqBy(
           [...(txOut.value.assets?.keys() || [])]
-            .filter((assetId) => handlePolicyIds.some((policyId) => assetId.startsWith(policyId)))
+            .filter((assetId) => {
+              const matchPolicyId = handlePolicyIds.some((policyId) => assetId.startsWith(policyId));
+              if (matchPolicyId) {
+                const assetName = Cardano.AssetId.getAssetName(assetId);
+                const decoded = Asset.AssetNameLabel.decode(assetName);
+                if (decoded && decoded.label !== Asset.AssetNameLabelNum.UserNFT) {
```

Comment:
`return !decoded || decoded.label === Asset.AssetNameLabelNum.UserNFT`

is a bit more compact and maybe more easy to read.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/905/commits/a1061aea9783682cf5b6f92e6ff0135e0eea7b40